### DISCUSSION TITLE
What are the best possible security settings?

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#3276] [Q&A]
I found this https://github.com/sumatrapdfreader/sumatrapdf/blob/master/docs/sumatrapdfrestrict.ini with some settings but I am not sure which settings I should edit to enable better security. Will some of these make no difference at all in terms of security?

Also At the bottom there are two settings 
LinkProtocols = http,https,mailto
SafeFileTypes = audio,video,webpage

It sounds like these would be the best ones to disable since it would block all types of links in the PDF file. Is this correct ?
To edit this what would I need to change the file to? Do I simply delete all the text after the = sign and leave it as 

LinkProtocols = 
SafeFileTypes = 

?

**Comment by kjk** [comment]
Those are not for security.

Those are meant for people who want to use Sumatra on a public computer (like a kiosk) and restrict what people can do.

You don't have to click any links in a PDF and there's really no security issue with opening links. You click random links in web pages all the time and it's fine.

**Reply by ghost** [reply]
There can be hidden links which I accidentally click, leading to bad websites or files. What if the links are hidden in a way which I cannot see them when viewing the PDF file, and accidentally clicking part of the PDF file will open the link. So this is why I thought it would be a good idea to disable the link settings

**Comment by GitHubRulesOK** [comment]
In a Home setting With younger "staff" you may wish to limit their exposure when reading PDF 
(will not stop older ones, but good for their learning) 
So by adding restrictions you could try to reduce their chances of going to the dark side.

Thus for those users and your chance of accidents this should be enough.
```
[Policies]
; Whether SumatraPDF should be allowed to access the Internet.
; Needed for:
; * Checking for updates
; * Sending crash reports
InternetAccess = 0
```
Alternately to see links without them active but still allow opening files you could use
```
[Policies]
; Whether SumatraPDF should be allowed to access the Internet.
; Needed for:
; * Checking for updates
; * Sending crash reports
InternetAccess = 1
DiskAccess = 1
LinkProtocols =
```

![image](https://user-images.githubusercontent.com/12754913/219248107-4b50d9b5-4730-461c-aec2-8c6cf50e6540.png)

**Reply by ghost** [reply]
Thats good thanks
